website/content/gateway/api_reference/transaction_status/transaction_status_codes_table.md

```diff
@@ -4,27 +4,29 @@ date: 2022-04-13T12:37:22+02:00
-| Declined | 40000 | General input error                | ✓    | ✓       | ✓      | ✓    | ✓      |
-|          | 40110 | Invalid card number                | ✓    |         |        |      | ✓      |
-|          | 40111 | Unsupported card scheme            | ✓    |         |        |      | ✓      |
-|          | 40120 | Invalid CSC                        | ✓    |         |        |      |        |
-|          | 40130 | Invalid expire date                | ✓    |         |        |      | ✓      |
-|          | 40135 | Card expired                       | ✓    |         |        |      | ✓      |
-|          | 40140 | Invalid currency                   | ✓    |         |        |      | ✓      |
-|          | 40150 | Invalid text on statement          | ✓    | ✓       | ✓      |      | ✓      |
-|          | 40200 | Clearhaus rule violation           | ✓    | ✓       | ✓      | ✓    | ✓      |
-|          | 40300 | 3-D Secure problem                 | ✓    |         |        |      |        |
-|          | 40310 | 3-D Secure authentication failure  | ✓    |         |        |      |        |
-|          | 40400 | Backend problem                    | ✓    |         |        | ✓    | ✓      |
-|          | 40410 | Declined by issuer or card scheme  | ✓    |         |        | ✓    | ✓      |
-|          | 40411 | Card restricted                    | ✓    |         |        |      | ✓      |
-|          | 40412 | Card lost or stolen                | ✓    |         |        |      | ✓      |
-|          | 40413 | Insufficient funds                 | ✓    |         |        |      | ✓      |
-|          | 40414 | Suspected fraud                    | ✓    |         |        |      | ✓      |
-|          | 40415 | Amount limit exceeded              | ✓    |         |        |      | ✓      |
-|          | 40416 | Additional authentication required | ✓    |         |        |      | ✓      |
-|          | 40420 | Merchant blocked by cardholder     | ✓    |         |        |      | ✓      |
-|          | 50000 | Clearhaus error                    | ✓    | ✓       | ✓      | ✓    | ✓      |
+| Code  | Meaning                            | Auth | Capture | Refund | Void | Credit | Debit  | Debit Refund |
+|-------|------------------------------------|------|---------|--------|------|--------|--------|--------------|
+| 20000 | Approved                           | ✓    | ✓       | ✓      | ✓    | ✓      | ✓      | ✓            |
+| 40000 | General input error                | ✓    | ✓       | ✓      | ✓    | ✓      | ✓      | ✓            |
+| 40110 | Invalid card number                | ✓    |         |        |      | ✓      | ✓      | ✓            |
+| 40111 | Unsupported card scheme            | ✓    |         |        |      | ✓      | ✓      |              |
+| 40120 | Invalid CSC                        | ✓    |         |        |      |        | ✓      | ✓            |
```

Comment:
Debit refunds can receive "Invalid CSC"? There should be no CSC sent, right? 🤨

Reply:
You are correct, fixed in [clearhaus/gateway-api-docs@`f734b37` (#172)](https://github.com/clearhaus/gateway-api-docs/pull/172/commits/f734b37e5390e22b2a37a8a2be93af4b59c3f4da)

Reply:
I was about to disagree, however, I see we do not currently map any responses from upstream to 40120 for debits and debit refunds.

But! We should start to perform such mapping. Especially for debits, where it indeed can make sense. Please await an internal code PR before moving on.

Reply:
With the internal code PR now created, I'll argue that both debits and debit refunds should have a :heavy_check_mark: for invalid CSC.

Reply:
But how should a client get an `Invalid CSC` triggered on a refund? A client can only refund a debit that has gone through this check and we use the csc value stored with the card. As a client you only send the debit id, right. :thinking: 

I agree that we could just add it there also because Visa could answer this to us.

Reply:
You write it yourself :slightly_smiling_face: 

> because Visa could answer this to us.

It is the same story as 40413 / insufficient funds, for which @ct-clearhaus commented here: https://github.com/clearhaus/gateway-api-docs/pull/172#discussion_r1182112969

Reply:
I do agree it is annoying we have to have all these checkmarks because issuers can potentially respond with all response codes. It almost ruins the value of the table.

Reply:
@ct-clearhaus Do you have any final input to this?

Reply:
I've reverted the before mentioned Debit refund Invalid CSC marking in [`b45b9b0` (#172)](https://github.com/clearhaus/gateway-api-docs/pull/172/commits/b45b9b03a8c3118158a9b5d33c59ecf638b293c8)

Reply:
You are simply awesome! 🤗